cmd/cmdtest.go

```diff
@@ -31,7 +31,12 @@ func CallCmd(
 	rootCmd.SetArgs(args)
 
 	err = rootCmd.Execute()
+	outcome := analytics.SUCCESS
```

Comment:
Do we need this here? I commented it out and the tests still pass.

Reply:
Looking into this

Reply:
I realized we don't need this variable anyway. But my main worry from this comment was that the tests weren't working properly. I double checked that the "CLI Command Completed" event must have the right outcome "success" or "error" to for the tests to pass.